java/client/src/main/java/glide/api/commands/ServerManagementCommands.java

```diff
@@ -32,4 +32,13 @@ public interface ServerManagementCommands {
      *     sections requested.
      */
     CompletableFuture<String> info(InfoOptions options);
+
+    /**
+     * Change the currently selected Redis database.
+     *
+     * @see <a href="https://redis.io/commands/select/">redis.io</a> for details.
+     * @param index - The index of the database to select.
```

Comment:
```suggestion
     * @param index The index of the <code>database</code> to select.
```

Reply:
why do you want to emphasize database?

Reply:
It is one the redis terms, as set (not a command name)

Reply:
We don't emphasize those terms.  Just Java terms.  Same goes for 'index'.  For Redis-based set, we wouldn't emphasize it either.  Although, we could...